Thanks @scholzj for the reminder. I checked other recent `acceptance` runs and none showed this timeout.

I went back through the `systemtest-logs` artifact from this PR's original failing run and found the actual error:

```
+ exec /usr/bin/tini -w -e 143 -- java ... -classpath :lib/io.strimzi.user-operator-1.1.0-SNAPSHOT.jar:@project.dist.classpath@ io.strimzi.operator.user.Main
Error: Unable to initialize main class io.strimzi.operator.user.Main
Caused by: java.lang.NoClassDefFoundError: io/strimzi/operator/common/AdminClientProvider
```

This is unrelated to the broker. The User Operator in the Entity Operator pod crashes on startup because the launch classpath still contains the literal `@project.dist.classpath@` placeholder, so `operator-common` never gets loaded and `AdminClientProvider` cannot be found.

Adding the Mockito javaagent to `user-operator/pom.xml` changed Maven package-phase plugin order: dist assembly now runs before `dependency:build-classpath`. The build passes, but the dist zip is broken. Only UO is affected because it was the only module where surefire was appended after `maven-assembly-plugin`.

@bboyleonp666 The Mockito javaagent config in `user-operator/pom.xml` appears to cause this. In that module, `maven-surefire-plugin` was added after `maven-assembly-plugin`, unlike the other operator modules. Moving it to the beginning of the `<plugins>` list should restore the correct `package-phase` order. Alternatively, moving `copy-dependencies` and `set-classpath` to `prepare-package` would also work and is a bit more robust. WDYT?

--- a/user-operator/pom.xml
+++ b/user-operator/pom.xml
@@ -136,6 +136,13 @@
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <argLine>@{argLine} -javaagent:${org.mockito:mockito-core:jar}</argLine>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-dependency-plugin</artifactId>
                 <version>${maven.dependency.version}</version>
                 <executions>
@@ -179,13 +186,6 @@
                         <phase>package</phase>
                     </execution>
                 </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <argLine>@{argLine} -javaagent:${org.mockito:mockito-core:jar}</argLine>
-                </configuration>
             </plugin>
         </plugins>
     </build>

